internal/syncer/connector/confluence_test.go

```
@@ -0,0 +1,318 @@
+		"page_id":           "123",
+		"index_recursively": true,
+		"batch_size":        10,
+	})
+	session, err := connector.OpenSync(context.Background(), SyncRequest{
+		FromBeginning: true,
+		WindowEnd:     confluenceTestTime(t, "2026-01-03T00:00:00Z"),
+	})
+	if err != nil {
+		t.Fatalf("OpenSync() error = %v", err)
+	}
+	defer session.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `session.Close` is not checked

(errcheck)